File: lua/test/exists_test.lua
-- Airtable SDK exists test

local sdk = require("airtable_sdk")

describe("AirtableSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Airtable SDK exists test

require_once __DIR__ . '/../airtable_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = AirtableSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
